pdr_backend/lake/etl.py

```diff
@@ -69,9 +70,11 @@ def __init__(self, ppss: PPSS, gql_data_factory: GQLDataFactory):
             bronze_pdr_slots_table_name: get_bronze_pdr_slots_data_with_SQL,
         }
 
+        print(f"self.bronze_table_getters: {self.bronze_table_getters}")
+
         self.bronze_table_names = list(self.bronze_table_getters.keys())
 
-        self.temp_table_names = [*self.bronze_table_names, *self.raw_table_names]
+        self.temp_table_names = [*self.bronze_table_names]
```

Comment:
you removed raw_table_names from the temp_table_names but we have move the temp data to the prod data tables when the etl process is completed

Reply:
Yes, please remember that raw_table_names will now have already moved from _temp -> prod inside of GQLDataFactory, before ETL kicks off.
![image](https://github.com/oceanprotocol/pdr-backend/assets/69865342/4812169f-3318-42ab-b0e0-0a93cc294c88)

So when ETL runs, the only temp_tables will be the bronze, silver, gold.